codes/utils/util.py

```diff
@@ -126,6 +127,36 @@ def save_img(img, img_path, mode='RGB'):
     cv2.imwrite(img_path, img)
 
 
+# Gaussian kernel for downsampling used in DUF official code
```

Comment:
In the original DUF repo, it supports x2, x3, and x4. Can we also support these?
And maybe we can also convert the x2, x3 modes, too.